resources/entrypoint.sh

```diff
@@ -2,7 +2,7 @@
 if [ -f "/var/www/html/occ" ]; then
     # Apply one-click-instance settings...
     sudo -u www-data php /var/www/html/occ config:system:set one-click-instance --value=true --type=bool
-    sudo -u www-data php /var/www/html/occ config:system:set one-click-instance.user-limit --value=100 --type=int
+    sudo -u www-data php /var/www/html/occ config:system:set one-click-instance.user-limit --value=500 --type=int
```

Comment:
Where was this requested? Should we do the same on AIO?

Reply:
Directly from Frank via mail

Reply:
@blizzz can you ping marketing then that the univention site (https://nextcloud.com/univention) needs updating? (500 is the limit not 100)